services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/prometheus_metrics.py

```diff
@@ -113,7 +113,7 @@ async def _update_metrics(self):
             )
             self._metrics_response = MetricsResponse.from_error(e)
         except Exception as e:  # pylint: disable=broad-exception-caught
-            _logger.exception("Unexpected exception")
+            _logger.info("Unexpected exception", exc_info=True)
```

Comment:
just use _logger.exception.

Reply:
I can't do that here, I want to use a lower log level than ERROR. That's why I've changed it from _logger.exception to this